packages/odyssey-react-mui/src/themes/odyssey/components.ts

```diff
@@ -146,10 +145,20 @@ export const components: ThemeOptions["components"] = {
           },
         },
       },
+      {
+        // icon only
+        props: { children: "" },
+        style: {
+          minWidth: "auto",
+
+          ".MuiButton-startIcon": {
+            margin: "0",
+          },
+        },
+      },
     ],
     styleOverrides: {
-      root: ({ ownerState }) => ({
-        ...ownerState,
```

Comment:
Do you mind making this change for other components where we are spreading the whole object in as well?